language/cloud-client/v1/snippets.py

```diff
@@ -39,45 +39,45 @@ def sentiment_text(text):
         text = text.decode('utf-8')
 
     # Instantiates a plain text document.
-    # [START migration_document_text]
-    # [START migration_analyze_sentiment]
+    # [START language_python_migration_document_text]
+    # [START language_python_migration_sentiment_text]
     document = types.Document(
         content=text,
         type=enums.Document.Type.PLAIN_TEXT)
-    # [END migration_document_text]
+    # [END language_python_migration_document_text]
 
     # Detects sentiment in the document. You can also analyze HTML with:
     #   document.type == enums.Document.Type.HTML
     sentiment = client.analyze_sentiment(document).document_sentiment
 
     print('Score: {}'.format(sentiment.score))
     print('Magnitude: {}'.format(sentiment.magnitude))
-    # [END migration_analyze_sentiment]
-# [END def_sentiment_text]
+    # [END language_python_migration_sentiment_text]
+# [END language_sentiment_text]
 
 
-# [START def_sentiment_file]
+# [START language_sentiment_file_gcs]
```

Comment:
should this be just the gcs suffix instead of file_gcs?

Reply:
To be consistent with the other samples, there is a `_gcs` suffix when the file is from GCS, and no suffix when it is a local file.